bedrock/base/static.py

```diff
@@ -12,11 +12,13 @@ class BedrockWhiteNoise(DjangoWhiteNoise):
     def __init__(self, application):
         super(BedrockWhiteNoise, self).__init__(application)
 
-        # TODO add tests for these in mcom-tests
+        # TODO add functional tests for these 
```

Comment:
flake8 doesn't like the trailing whitespace on the comment :( seems a bit picky, but okay.